[
  {
    "type": "value_error",
    "loc": [
      "contributor",
      2,
      "Person"
    ],
    "msg": "Value error, Contact person must have an email address.",
    "input": {
      "name": "Orazem, Mark",
      "roleName": [
        "dcite:ContactPerson"
      ],
      "schemaKey": "Person",
      "identifier": "0000-0003-3668-7767",
      "includeInCitation": true
    },
    "ctx": {
      "error": "Contact person must have an email address."
    },
    "url": "https://errors.pydantic.dev/2.10/v/value_error"
  }
]